src/Symfony/Component/Security/Guard/Firewall/GuardAuthenticationListener.php

```diff
@@ -11,16 +11,17 @@
 
 namespace Symfony\Component\Security\Guard\Firewall;
 
+use Psr\Log\LoggerInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Event\GetResponseEvent;
-use Symfony\Component\Security\Guard\GuardAuthenticatorHandler;
-use Symfony\Component\Security\Guard\Token\PreAuthenticationGuardToken;
 use Symfony\Component\Security\Core\Authentication\AuthenticationManagerInterface;
-use Symfony\Component\Security\Guard\GuardAuthenticatorInterface;
-use Psr\Log\LoggerInterface;
 use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
 use Symfony\Component\Security\Core\Exception\AuthenticationException;
+use Symfony\Component\Security\Guard\GuardAuthenticatorHandler;
+use Symfony\Component\Security\Guard\Authenticator\GuardAuthenticatorInterface;
+use Symfony\Component\Security\Guard\GuardAuthenticatorInterface as GuardAuthenticatorLegacyInterface;
+use Symfony\Component\Security\Guard\Token\PreAuthenticationGuardToken;
```

Comment:
Unless there's a standard I don't know of, can you move the `use` statements back to their original spot? It's nit-picky I know, but it will help avoid conflicts in the future.